OptimoveSDK/Sources/Classes/Optimobile/OptimoveInApp.swift

```diff
@@ -4,49 +4,53 @@ import CoreData
 import Foundation
 import OptimoveCore
 
-class InAppInboxItem {
+final class InAppInboxItem {
     private(set) var id: Int64
     private(set) var title: String
     private(set) var subtitle: String
     private(set) var availableFrom: Date?
     private(set) var availableTo: Date?
     private(set) var dismissedAt: Date?
     private(set) var sentAt: Date
-    private(set) var data: NSDictionary?
+    private(set) var data: ObjcJSON?
```

Comment:
Are these 'breaking' API changes?

Reply:
Great, all for these kind of things in general, Codable will be simpler, but I think this model `InAppInboxItem` is not the Core Data entity, it is the model we expose to consumers, so changing this field could be breaking/leaky.

Reply:
Hmm, this class cannot be exposed to a consumer -- it is not public